testng-curl/browserstack.yml

```diff
@@ -0,0 +1,72 @@
+# =============================
+# Set BrowserStack Credentials
+# =============================
+# Add your BrowserStack userName and accessKey here or set BROWSERSTACK_USERNAME and
+# BROWSERSTACK_ACCESS_KEY as env variables
+userName: ${BROWSERSTACK_ACCESS_KEY}
+accessKey: ${BROWSERSTACK_USERNAME}
+
+# ======================
+# BrowserStack Reporting
+# ======================
+# The following capabilities are used to set up reporting on BrowserStack:
+# Set 'projectName' to the name of your project. Example, Marketing Website
+projectName: XYZ Banking Corporation
+# Set `buildName` as the name of the job / testsuite being run
+buildName: Test Run
+# `buildIdentifier` is a unique id to differentiate every execution that gets appended to
+# buildName. Choose your buildIdentifier format from the available expressions:
+# ${BUILD_NUMBER} (Default): Generates an incremental counter with every execution
+# ${DATE_TIME}: Generates a Timestamp with every execution. Eg. 05-Nov-19:30
+# Read more about buildIdentifiers here -> https://www.browserstack.com/docs/automate/selenium/organize-tests
+buildIdentifier: '${DATE_TIME} #${BUILD_NUMBER}' # Supports strings along with either/both ${expression}
+# Set `framework` of your test suite. Example, `testng`, `cucumber`, `cucumber-testng`
+# This property is needed to send test context to BrowserStack (test name, status)
+framework: testng
```

Comment:
Do we really need code from Line 9 - Line 25 ?
These is used by Observability team and not us, and have already mentioned earlier, this is a cURL folder where we should not add any Observability related code